misc/toml-parser/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -0,0 +1,228 @@
+{
+    "root": "Node",
+    "type": "abstract",
+    "attributes": [
+        {
+            "name": "kind",
+            "type": "SyntaxKind"
+        },
+        {
+            "name": "width",
+            "type": "int"
+        }
+    ],
+    "nodes": [
+        {
+            "name": "ModulePartNode",
+            "base": "Node",
+            "kind": "MODULE_PART",
+            "attributes": [
+                {
+                    "name": "members",
+                    "type": "ModuleMemberDeclarationNode",
+                    "occurrences": "MULTIPLE"
+                },
+                {
+                    "name": "eofToken",
+                    "type": "Token"
+                }
+            ]
+        },
+        {
+            "name": "ModuleMemberDeclarationNode",
+            "base": "Node",
+            "isAbstract": true
+        },
+        {
+            "name": "ValueNode",
+            "base": "Node",
+            "isAbstract": true
+        },
+        {
+            "name": "TableNode",
+            "base": "ModuleMemberDeclarationNode",
+            "kind": "TABLE",
+            "attributes": [
+                {
+                    "name": "openBracket",
+                    "type": "Token",
+                    "isOptional": false
```

Comment:
We need this `isOptional` flag, only if its set to `true`. Otherwise we can remove those